src/wifi_manager.c

```diff
@@ -381,6 +381,16 @@ wifi_manager_start(
     return res;
 }
 
+void
+wifi_manager_cb_on_user_req(const http_server_user_req_code_e req_code)
+{
+    if (NULL == g_wifi_callbacks.cb_on_http_user_req)
+    {
+        return;
+    }
+    return g_wifi_callbacks.cb_on_http_user_req(req_code);
```

Comment:
return statement on void function is a bit confusing, maybe this would be clearer?
```
g_wifi_callbacks.cb_on_http_user_req(req_code);
return;
```

Reply:
> return statement on void function is a bit confusing, maybe this would be clearer?
> 
> ```
> g_wifi_callbacks.cb_on_http_user_req(req_code);
> return;
> ```

thanks, you're right. unfortunately gcc does not show a warning in this case